FlthyHPs_1.2

```diff
@@ -422,13 +426,14 @@ int functionState;     // Holds variable that identifies the function within the
 int optionState;       // Holds variable that identifies the option value for those commands that use it. (color, position, etc)
 int optionState2;
 
-byte HP_command[HPCOUNT][2] = {{0,0},
-                               {0,0},
-                               {0,0}};
-
-byte LED_command[HPCOUNT][3] = {{0,0,0},
-                                {0,0,0},
-                                {0,0,0}};
+#define HPCOMMANDLEN 2
+byte HP_command[HPCOUNT][HPCOMMANDLEN] = {{0,0},
```

Comment:
a tidier way of doing this is to create a typdef struct for your commands:
eg:
```
typedef struct LEDCmdStruct 
{
   byte LEDFunction;
   byte LEDOption1;
   byte LEDOption2;
} LEDCmd;
```
you then can declare an array of these types instead of an array of of array of bytes.
eg:
`LEDCmd LED_command[HPCOUNT];`
Logically the memory storage is identical to the way you have but now your code can be easier to read because instead of having to remember that subscript1 is option1 and subscript2 is option2 the code would look like this:
```
LED_command[0].LEDFunction = functionState;
LED_command[0].LEDOption1 = optionState;
```
And clearing the memory looks easier too because you just do:
`memset(LED_command[0], 0, sizeof(LEDCmd));`
(and even nicer, but I haven't tried this yet on arduino is: LED_command[0] = (LEDCmd) {0};